### DISCUSSION TITLE
After upgrading Version from 1.10.2 to 1.10.14 there is issue with Code of DAG

### DISCUSSION THREAD

**Discussion by rk030912** [apache/airflow#22482] [Q&A]
### Apache Airflow version

2.0.0

### What happened

After upgrading Version from 1.10.2 to 1.10.14 there is issue with Code of DAG, it behaves weird when i am checking the code in the Airflow UI for DAG1 , it is showing code of DAG2, when i refresh it it again shows the original code, but when i refresh it again it showing some other DAG code, is this a bug in Airflow 1.10.14?

### What you think should happen instead

Please explain why is this behaviour and suggest any solution

### Operating System

Ubuntu

### Deployment

Docker-Compose

### Anything else

every DAG refresh in the Airflow UI

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by eladkal** [comment]
Airflow 1.10.14 is EOL - even if there is a bug we can't do anything about it.
I suggest to update to latest Airflow version as soons as possible.
